docs/asgi.rst

```diff
@@ -17,7 +17,9 @@ Summary
 -------
 
 An ASGI application is a callable that takes a scope and returns a coroutine
-callable, that takes receive and send methods. It's usually written as a class::
+callable, that takes receive and send methods. It's usually written as a class:
+
+.. code-block:: python
```

Comment:
Python is the default right?

Reply:
It is but I think using `::` all over the place makes it hard for copy-pasters to find out about `code-block` and thus realize they can control the language highlighting. I prefer to be explicit

Reply:
Yeah, OK.